### PR TITLE
Fix Cert invalid issue in chrome for mac

### PR DESCRIPTION
**Pull request by ashans**

### CODE REVIEW

File: samples/asgardeo-html-js-app/package.json [hunk 1]
```diff
@@ -20,9 +20,11 @@
     "license": "Apache-2.0",
     "private": false,
     "scripts": {
-        "start": "http-server -p 5000 -S -o -a localhost"
+        "generateCertificates": "npx openssl req -config openssl.cnf -new -sha256 -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 30 -out cert.pem",
```
Comment (brionmario) [inline samples/asgardeo-html-js-app/package.json:23]: Shall we rename the script to `generate:ssl-cert`? WDYT?
```suggestion
        "generate:ssl-cert": "npx openssl req -config openssl.cnf -new -sha256 -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 30 -out cert.pem",
```

File: samples/asgardeo-html-js-app/package.json [hunk 2]
```diff
@@ -20,9 +20,11 @@
...
+        "start": "npm run generateCertificates && http-server -p 5000 -S -o -a localhost"
```
Comment (brionmario) [inline samples/asgardeo-html-js-app/package.json:24]:
```suggestion
        "start": "npm run generate:ssl-cert && http-server -p 5000 -S -o -a localhost"
```

**Review by brionmario — APPROVED** [review]

File: samples/asgardeo-html-js-app/package.json (hunk 2 above)
Comment (ashans) [reply samples/asgardeo-html-js-app/package.json]: Removed in bb53963

File: samples/asgardeo-html-js-app/package.json (hunk 1 above)
Comment (ashans) [reply samples/asgardeo-html-js-app/package.json]: Removed in bb53963

**Pull request merged**